docs/UsersGuide/source/Reference/RocotoInfo.rst

```diff
@@ -138,35 +143,35 @@ from the ``$EXPTDIR`` directory as follows:
 
 .. code-block:: console
 
-   rocotocheck -w </path/to/workflow/xml/file> -d </path/to/workflow/database/> file -c <YYYYMMDDHHmm> -t <taskname> 
+   rocotocheck -w FV3LAM_wflow.xml -d FV3LAM_wflow.db file -c <YYYYMMDDHHmm> -t <taskname> 
 
 where 
 
-* ``-c`` is the cycle to query in YYYYMMDDHHmm format
-* ``-t`` is the task name (e.g., ``make_grid``, ``get_extrn_ics``, ``run_fcst``). 
+* ``-c`` is the cycle to query in YYYYMMDDHHmm format.
+* ``-t`` is the task name (e.g., ``make_grid``, ``get_extrn_ics``, ``run_fcst_mem000``). 
 
-The cycle and task names appear in the first and second columns of the table output by ``rocotostat``. 
+The cycle and task names appear in the first and second columns of the table output by ``rocotostat``. Users will need to include the absolute or relative path to the worflow XML and database files when running the command from another directory.
```

Comment:
workflow rather than worflow.
```suggestion
The cycle and task names appear in the first and second columns of the table output by ``rocotostat``. Users will need to include the absolute or relative path to the workflow XML and database files when running the command from another directory.
```